docs/Vector-Similarity-for-RediSearch.ipynb

```diff
@@ -37,40 +37,96 @@
     "\n",
     "Command format:\n",
     "```\n",
-    "FT.CREATE <index_name> SCHEMA <vector field name> VECTOR <type> <dimension> <distance metric> <index algorithm> <algorithm parameters>\n",
+    "FT.CREATE <index_name> SCHEMA <vector field name> VECTOR <type> <num params> [<algorithm parameter name> <parameter value> ...]\n",
     "```\n",
     "### General indexing mandatory parameters\n",
     "Parameters should be given to the index build command in the following order\n",
     "* type - vector data type -  - Currently only `FLOAT32` is supported\n",
     "* dimension - vector dimension.\n",
     "* distance metric - either `L2` for euclidean distance, `IP` for internal product or `COSINE` for cosine similarity should be provided. Note, when `COSINE` is selected the indexed vectors will be normalized upon indexing, and the query vector will be normalized upon query.\n",
```

Comment:
One of: L2, IP or COSINE

Reply:
(not my explanation but) here we explain what does the short names mean, later we simply say `**DISTANCE_METRIC** - Supported distance metric. Currently one of **{`L2`, `IP`, `COSINE`}**`